article/app/views/fragments/articleHeaderGarnett.scala.html

```diff
@@ -96,23 +96,24 @@ <h1 class="content__headline @if(article.content.starRating && !article.elements
                 }
 
                 @article.content.imdb.map { imdbId =>
-                    <div articleprop="articleReviewed" articlescope articletype="http://schema.org/Movie">
-                        <link articleprop="sameAs" href="http://www.imdb.com/title/@imdbId/">
+                    <div itemprop="itemReviewed" itemscope itemtype="http://schema.org/Movie">
+                        <link itemprop="sameAs" href="http://www.imdb.com/title/@imdbId/">
                         @defining(article.content.primaryKeyWordTag.map(_.name).getOrElse(".")) { tag =>
                             @* we're not the authority on the film name, but just to keep google validator happy
-                            *@<meta articleprop="name" content="@tag"/>
+                            *@<meta itemprop="name" content="@tag"/>
+                            <meta itemprop="image" content="@article.content.openGraphImage">
                         }
                     </div>
                 }
 
                 @article.content.isbn.map { isbn =>
-                    <div articleprop="articleReviewed" articlescope articletype="http://schema.org/Book">
-                        <meta articleprop="isbn" content="@isbn">
-                        <div articleprop="author" articlescope articletype="http://schema.org/Person">
-                            <meta articleprop="sameAs" content="http://schema.org/Person@* we can't know *@">
-                            <meta articleprop="name" content=".@* we can't know *@">
+                    <div itemprop="itemReviewed" itemscope itemtype="http://schema.org/Book">
```

Comment:
I haven't tried to fix books completely here, just fixed the obviously wrong stuff (article* instead of item*).